fvtr/python/python.exp

```diff
@@ -105,6 +105,10 @@ set env(TMPDIR) "$FULLPATH/tmp[pid]"
 exec mkdir $env(TMPDIR)
 printit "TMPDIR=\"$env(TMPDIR)\""
 
+# Test for the issue #45
+printit "Executing sqlite3 importing test."
+exec $env(AT_DEST)/bin/python3 -c "import sqlite3"
+
```

Comment:
I've just found an error here: python3 is not available on all AT versions.  If we decide to use this test instead of reusing test_sqlite from the Python testsuite, this line has to be changed.